dell-restore-system

```diff
@@ -0,0 +1,89 @@
+#! /usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Dell Recovery Restore system command line
+#
+# Copyright (C) 2017 Dell Inc,
+#   Author: Mario Limonciello/Jude Hung
+#
+# This is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 2 of the License, or at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this application; if not, write to the Free Software Foundation, Inc., 51
+# Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+##################################################################################
+
+from Dell.recovery_common import (find_partition, check_version, DBUS_INTERFACE_NAME,
+                                  DBUS_BUS_NAME, dbus_sync_call_signal_wrapper,
+                                  PermissionDeniedByPolicy)
+import optparse
+import os
+import subprocess
+import sys
+#Translation support
+from gettext import gettext as _
+
+usage = '%prog [options]'
+parser = optparse.OptionParser(usage=usage)
+parser.set_defaults(
+    version=''
+    )
+parser.add_option('-c', '--check-version', dest='checkversion', action='store_true',
+                  help='Show the version information.')
+parser.add_option('-y', '--yes', dest='yes', action='store_true',
+                  help="Restore the system without user's confirmation.")
+(options, args) = parser.parse_args()
+
+if __name__ == '__main__':
+    if options.checkversion:
+        print("Version: %s" % check_version())
```

Comment:
Do you want this string translated?  You should invoke with _ if so.

Reply:
Sure, I could update "Version:" so it could be supported for i18n.